### PR TITLE
Remove file button added

### PR DESCRIPTION
**Pull request by NitinBhasneria**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

editor1

78994733-6c292700-7b5e-11ea-95be-64467d7d1ae8

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by NitinBhasneria** [comment]
@VladimirMikulic Have a look. Thanks.

**Comment by VladimirMikulic** [comment]
How many of these "remove file button" PRs do we have? I feel like you spawn each one every day?

**Comment by NitinBhasneria** [comment]
Nice joke @VladimirMikulic. There is only two PR's I have closed the previous as according to you it was messy. And that why I have open the clean PR of that. Have a look, also you should no talk like this.

**Comment by VladimirMikulic** [comment]
"Only" 2 closed PRs? You shouldn't close a PR and open a new one every time someone suggests the changes. That's highly anti-pattern and discouraged. No PR floods. You push a commit that includes improvements to the existing PR, we later squash commits and that's it.

My job is to point out the mistakes and I can't approve the PR if it's not ready.
If you want a faster PR approval, then thoroughly test the changes before you submit the PR, ask yourself "Does my code follow the best practices?", "Is code style alright?", "Is the code easily understandable?" etc.

Also, there is the @publiclab/reviewers team. You don't have to "prefer" anyone.
By "preferring" someone in every single PR, know that the person you tagged needs to go through the issue, trough your code and potentially suggest/approve changes, which takes time.

File: src/modules/PublicLab.MainImageModule.js [hunk 1]
```diff
@@ -139,8 +142,17 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
 
       }
 
+      
     });
 
+    // Remove Image button
+    var mainImage = document.getElementById("mainImage");
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:149]: Why double quotes here when the whole file uses single quotes?

File: src/modules/PublicLab.MainImageModule.js [hunk 2]
```diff
@@ -139,8 +142,17 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
+    var removeFile = document.getElementById("removeFile");
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:150]: Here, again.

File: src/modules/PublicLab.MainImageModule.js [hunk 3]
```diff
@@ -139,8 +142,17 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
+    removeFile.onclick = function() {
+      mainImage.style.background = "white";
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:152]: As well as here.

File: src/modules/PublicLab.MainImageModule.js [hunk 4]
```diff
@@ -139,8 +142,17 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
 
       }
 
+
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:145]: Any reason for an additional blank line?

**Comment by NitinBhasneria** [comment]
Yea, so you can say me this. Not you have to keep on anger. I will fix this today itself.

**Comment by VladimirMikulic** [comment]
@NitinBhasneria I saw your edit comment and you misunderstood me.
I am not angry, I am just suggesting the changes that you need to make to get PR merged.
I don't have the slightest idea why @sashadev-sky deleted my comment, hopefully, you had a chance to read it in your inbox.

The point was, don't open a PR every time someone suggests the changes. (highly anti-pattern)
Just push a commit that includes improvement and that's it, no hard feelings.

**Comment by NitinBhasneria** [comment]
Sorry for that @VladimirMikulic. Will not do so now onwards and thanks for the guidance.

**Comment by VladimirMikulic** [comment]
You don't have to apologise. Don't feel bad when you are requested to improve the code.
After all, that's why we use Github and VCS, right?

**Comment by NitinBhasneria** [comment]
Yea @VladimirMikulic Thanks

**Comment by NitinBhasneria** [comment]
@jywarren PLease have a look.

**Comment by NitinBhasneria** [comment]
@emilyashley  have a look on this.

**Comment by jywarren** [comment]
Hi all, sorry I missed this, and i'll check with @sashadev-sky -- perhaps it was a mistake -- thank you for working through this and I know sometimes online it's hard to be sure your tone is coming through in a friendly way, so I appreciate @NitinBhasneria for speaking up when you felt something and @VladimirMikulic for all your help on this PR. Small conflicts often happen when we are doing a lot of commenting/interacting very quickly and your patience is always appreciated. It's ALWAYS worth the effort to be sure our fellow community members are feeling welcome and supported. Our [code of conduct](http://publiclab.org/conduct) covers really bad behaviors, but it's the small kindnesses that often get us through our days feeling good. 

If you would like to discuss further please reach out to me, and I'm always available to listen and offer input -- thanks again for all your hard work. ❤️

**Comment by jywarren** [comment]
🎉

**Comment by keshav234156** [comment]
@NitinBhasneria Test are failing due to this PR. Please refine it asap.

**Comment by NitinBhasneria** [comment]
@keshav234156 It is failing due to #437 as the HTML file in tests is not containing the `mainImage` id.
Thanks

**Pull request merged**